drake/core/Path.h.in

```diff
@@ -2,7 +2,7 @@
 
 #include <string>
 
-namespace Drake {
+namespace {
```

Comment:
I would veto leaving an anonymous namespace in a header myself.  It is too easy for things to go amiss.